docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.fonts.scss

```diff
@@ -89,21 +89,28 @@
   }
 }
 
+@mixin hb-link--hover-style($box-shadow-color, $border-color) {
+  text-decoration: none;
+  margin-bottom: 0.0625em;
+  transition: box-shadow 100ms, border-bottom-color 0s;
+  transition-delay: 0s, 100ms;
+
+  &:hover,
+  &:focus {
+    box-shadow: inset 0 -0.5625em 0 $box-shadow-color;
+    transition-delay: 0s, 0s;
+    border-bottom-color: $border-color;
+  }
+}
+
 @mixin hb-link {
```

Comment:
I'm seeing a different link style on headings. This is what I see.
<img width="369" alt="heading" src="https://user-images.githubusercontent.com/12678977/73499700-cf445380-438e-11ea-886c-c24be0087ecd.png">

I _think_ we need to include this mixin on `hb-card__title` headings with links. That would affect all cards that have headings with links. Let's look at this together just to make sure. 👀